assets/sass/okta-theme.scss

```diff
@@ -820,3 +820,33 @@
     /* color: #f3f3f3; */
   }
 }
+
+/*  Remove the background image and borders on a smaller window size */
+@include max-width-mq(600px) {
+  // scss-lint:disable ImportantRule
+  .login-bg-image {
+    background-image: none !important;
+    background-color: #fff !important;
+    filter: unset !important;
+  }
+
+  /* See line 59. Unfortunately, we have to use the selector this way to override at small screen sizes. */
```

Comment:
nit: Might be just enough to say we use the same selector than in line 59?

Reply:
Haha okay!

Reply:
Changed to this: Using the same selector as line 59 to override styles at small screen sizes.

Reply:
sounds good. Once you push your changes this is ready, but let me know if you need me to review again.